utils/librarypath/librarypath.go

```diff
@@ -39,6 +40,25 @@ func cleanPath(paths []string, fullPath bool) []string {
 	return processedPaths
 }
 
+func ExistsInArray(array []string, item string) bool {
+	for _, value := range array {
+		if strings.Contains(strings.ToLower(item), strings.ToLower(value)) {
```

Comment:
Using `contains` in this case invalidate paths:
```go
{
  "ignorePath": [
	"vendor"
  ]
}
```

To path `./vendors` is valid but the path `./src/vendor/vendorCheck.pas` is ignored

Reply:
Do you have any solution for this problem? And if we put back slash before and after the path name in the boos.json file. Can you see any problems?

```
{
  "ignorePath": [
	"\vendor\"
  ]
}
```

Reply:
I think using strings.HasPrefix is better for now, based on the relative path of the project folder